src/App.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
-import React, { Component } from 'react';
-import {connect} from 'react-redux'
+import React, {Component} from 'react';
```

Comment:
Ha, my linter tells me to add spaces after the brackets.

Reply:
hilariously, that was the default that my `prettier` sublime plugin preferred and i reverted it because it looked funny to me, but i'm happy to go back